ocs_ci/ocs/ui/views.py

```diff
@@ -1047,6 +1051,11 @@
     "submariner-custom-channel": ("channel", By.ID),
 }
 
+acm_configuration_4_16 = {
+    "disaster-recovery": ("Disaster recovery", By.LINK_TEXT),
+    "policies": ("//a[normalize-space()='Policies']", By.XPATH),
```

Comment:
These locators should already be available in previous versions

Reply:
can u please point me out

Reply:
Couldn't find it, may be it changed. Ignore